lib/refresh.js

```diff
@@ -590,4 +588,19 @@ const cycle = function($page) {
   return promise;
 };
 
+function arrayShallowEqual (a, b) {
```

Comment:
Hmm, I can think of a few cheeky tricks because we know they are both arrays of strings, like `a.join(' ') === b.join(' ')` would work as long as we are sure the items don't contain spaces, but I'm not sure it is an improvement.

Reply:
Interesting alternative. Could join on some odd character like the ASCII Group Separator.

This looked like code that was trying to decide if dragging a page was a rearrangement that required further action. I think the only way to do this reliably in the presence of duplicated pages is to look up the code that represents the panel that holds a page and compare them. I will look into this as I find time today.